src/euphorie/client/browser/templates/preferences.pt

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xmlns:meta="http://xml.zope.org/namespaces/meta"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      id="new-email"
+      meta:interpolation="true"
+      metal:use-macro="context/@@shell/macros/shell"
+      tal:define="
+        client nocall:request/client;
+        webhelpers nocall:context/@@webhelpers;
+      "
+      i18n:domain="euphorie"
+>
+  <body>
+    <metal:content fill-slot="content"
+                   tal:define="
+                     base_url context/absolute_url;
+                   "
+    >
```

Comment:
Unless this is used by some macro I do not see the variable you define here used anyhwere:
```suggestion
    <metal:content fill-slot="content">
```